core/src/main/java/org/mobicents/ha/javax/sip/LoadBalancerHeartBeatingServiceImpl.java

```diff
@@ -639,7 +639,7 @@ protected void updateConnectorsAsSIPNode() {
 						if(jvmRoute != null) node.getProperties().put("jvmRoute", jvmRoute);
 						
 						node.getProperties().put("version", System.getProperty("org.mobicents.server.version", "0"));
-						
+						node.getProperties().put("sessionId", ""+System.currentTimeMillis());
 						if(gracefullyShuttingDown) {
```

Comment:
this won't work as everytime a connector is added or removed the sessionId will change as the nodes will be recomputed. The sessionID should be created once when the class is initialized and then reuse always the same id and not calling System.currentTimeMillis() every time. Did you test that solution ?